examples/portfolio/src/pages/index.astro

```diff
@@ -203,23 +216,21 @@ const featuredProject = projects[0];
 				<h1 class="title">
 					<small class="subtitle">The personal site of</small>Jeanine White
 				</h1>
-				<div>
-					<span class="role">👩‍💻 Developer <span class="invert">👩‍💻 Developer</span></span>&nbsp;
-					<span class="role">🎤 Speaker <span class="invert">🎤 Speaker</span></span>&nbsp;
+				<div class="roles">
+					<span class="role">👩‍💻 Developer <span class="invert">👩‍💻 Developer</span></span>
+					<span class="role">🎤 Speaker <span class="invert">🎤 Speaker</span></span>
 					<span class="role">✏️ Writer <span class="invert">✏️ Writer</span></span>
 				</div>
 				<p class="desc">Lover of dogs, roadtrips, and poetry.</p>
 			</div>
 		</header>
-		<main class="wrapper mt4 mb4">
+		<main class="wrapper">
 			<div class="grid">
 				<div class="section">
 					<h3 class="sectionTitle">Selected Work</h3>
 					<PortfolioPreview project={featuredProject} />
-					<div class="tac mt4">
-						<a href="/projects">
-							<Button>View All</Button>
-						</a>
+					<div class="buttonContainer">
+						<a href="/products" class="button">View All</a>
```

Comment:
```suggestion
						<a href="/products/" class="button">View All</a>
```
I don't know whether it matters here, but in Docs, we use trailing slashes in our links, or Hippo's link checker yells at us.